_posts/2020-05-27-ols1-wrapup.md

```diff
@@ -54,15 +54,15 @@ Last but not least, thanks to our expert community and speakers for sharing your
 ## Presentations - recap and common themes
 
 Let us recap it for you - we ran our first cohort from January to May 2020 with {{ all-participants | size }} mentees, {{ all-mentors | size }} mentors, and over 50 experts from Asian, African, European, Latin American, North American, and Russian countries.
-Our program introduced our mentees to a great deal of Open Science concepts that are directly applicable to their work. In addition, they also learned all about designing their project for inclusivity, establishing connections with others in the program, managing their communities, and working collaboratively.
-Through this program, OLS helped design and implement projects to solve technical and cultural questions by collaboratively designing open source software, AI techniques, educational resources, culture movements, and global partnerships in their communities ([details]({% link openseeds/ols-1/projects-participants.md %})).
+Our program introduced our mentees to a great deal of Open Science concepts that are directly applicable to their work. In addition, they also learned all about designing their project for inclusivity, establishing connections with others in the program, managing their communities, and working oratively.
```

Comment:
Why all `col` have been removed in this post?

Reply:
😳😳 Woah!

This is one of the HTML proofer error messages:
```
- ./_site/posts/2020/05/27/ols1-wrapup/index.html
  *  linking to internal hash #collaborators that does not exist (line 213)
     <a href="/ols-2#collaborators"><em>The Turing Way</em></a>
```

I did `CTRL+D` in vscode on the word "collab" to try and find the faulty internal hash. I did not realise that I removed all the instances of the word.

I'll fix that right away. Thank you for spotting this error.